Code/autopkglib/github/__init__.py

```diff
@@ -20,6 +20,7 @@
 import os
 import re
 import tempfile
+from urllib.parse import quote
```

Comment:
This whole file should disappear in a rebase, because I think this was merged in when you rebased dev onto master.

Reply:
Yes, I just didn't want to make the merge PR muddied with this bugfix PR. I'll pull and repush and everything will look much nicer.